tests/istio_test.go

```diff
@@ -38,3 +37,41 @@ func TestIstioClusterNode(t *testing.T) {
 func TestIstioDestinationRuleNode(t *testing.T) {
 	testNodeCreationFromConfig(t, istio.Manager, "destinationrule", objName+"-destinationrule")
 }
+
+func TestBookInfoScenario(t *testing.T) {
+	testRunner(
+		t,
+		[]Cmd{
+			{"kubectl apply -f /usr/bin/istio-1.0.1/samples/bookinfo/networking/destination-rule-all.yaml", true},
+		},
+		[]Cmd{
+			{"kubectl apply -f /usr/bin/istio-1.0.1/samples/bookinfo/platform/kube/cleanup.sh", false},
```

Comment:
same as above only you may consider extracting the base url to a constant or providing a utility function to generate the full url.

Reply:
DONE

Reply:
DONE